pkg/runtime/runtime-manifest.go

```diff
@@ -100,7 +100,15 @@ func resolveCredential(cd manifest.CredentialDefinition) (string, error) {
 }
 
 func (m *RuntimeManifest) resolveBundleOutput(def manifest.OutputDefinition) (string, error) {
-	// Get the output's value from the injected parameter source
+	// The output value may be tracked on the runtime manifest for use during the same action,
```

Comment:
Ah, I also think I prefer _a_ and can't currently think of an edge case... will work on this now.

Reply:
Ok.  Actually, option b seemed the most straightforward approach when I revisited this.  We do currently set step outputs last (https://github.com/deislabs/porter/blob/main/pkg/runtime/runtime.go#L98-L112).  In this case, we'll have set the value of a step output (that also happens to be a bundle output) during a previous step and so when we go to build the source data for a subsequent step (during the same action), we can defer to this value instead of attempting to override with the bundle-level wiring value (which will be empty since we're still executing the same action).  WDYT?